src/tribler/gui/i18n/es_ES.ts

```diff
@@ -1123,32 +1123,32 @@ Si no está seguro, pulse &quot;No&quot;. Más adelante podrá eliminar esas car
     <message>
         <location filename="../utilities.py" line="157"/>
         <source>%(years)iy %(weeks)iw</source>
-        <translation>%(años)iy %(semanas)iw</translation>
+        <translation>%(years)ia %(weeks)is</translation>
     </message>
     <message>
         <location filename="../utilities.py" line="159"/>
         <source>%(weeks)iw %(days)id</source>
-        <translation>%(semanas)iw %(días)id</translation>
+        <translation>%(weeks)is %(days)id</translation>
```

Comment:
**nitpick:** Currently, both weeks (semanas) and seconds (segundos) are abbreviated as `s`. I suspect it can cause some confusion. [According to the Wikipedia](https://en.wikipedia.org/wiki/Date_and_time_notation_in_Spain), in Spain, "the letters "m" and "s" are used to denote minutes and seconds". With this, I'd prefer to leave "weeks" without an abbreviation:
```suggestion
        <translation>%(weeks)i semanas %(days)i días</translation>
```
But we can leave it as is as well.